inference-engine/tests/functional/plugin/cpu/single_layer_tests/interpolate.cpp

```diff
@@ -132,7 +119,15 @@ std::vector<CPUSpecificParams> filterCPUInfoForDevice() {
     }
     return resCPUParams;
 }
-/* ========== */
+
+std::vector<fusingSpecificParams> getFusingPatterns() {
+    std::vector<fusingSpecificParams> fusingPatterns;
+    fusingPatterns.push_back(emptyFusingSpec);
+    if (with_cpu_x86_sse42()) {
+        fusingPatterns.push_back(fusingSubtractPerChannel);
```

Comment:
1) Just didn't get why we need this condition? We don't have HW w/o sse42 support in validation.
2) I think we need to extend list of post ops here.

Reply:
extended